### PR TITLE
Consolidating information on getting values from config and crosslinking it.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: docs/configuration/introduction.md [hunk 1]
```diff
@@ -58,18 +58,30 @@ Configuration Files
...
+| `app/config/routing.yml`      | The file where you can define custom urls for your website.
+| `app/config/permissions.yml`  | Here you can specify groups, users, etc. For most websites, the default permissions settings will be just fine.
 
-### Different configs per environment
+There are two other locations where configuration files can be found:
+
+| Folder                   | Description |
+| ------------------------ | ----------- |
+| `app/config/extensions/` | If you install extensions, their config files will be located in this directory.
+| `public/theme/`          | In the folder for the active theme, there can optionally be a `config.yml`.
```
Comment (xiaohutai) [inline docs/configuration/introduction.md]: Shouldn't this be `theme.yml`?

Comment (bobdenotter) [reply docs/configuration/introduction.md]: @xiaohutai Yes, thank you! :-)

**Comment by romulo1984** [comment]
@bobdenotter @GawainLynch 
The excerpt from [this line](https://github.com/bolt/docs/pull/857/files#diff-584be85325574fca04c386e46ea2b4c5R136)
 where it says it works with `theme.yml` is wrong. I submitted this PR, and would send another one to change. `theme.yml` does not accept environment variables.

[Discussed in this issue](https://github.com/bolt/bolt/issues/7177)

**Comment by GawainLynch** [comment]
Yeah, feel free to PR removing that bit :+1:

**Comment by romulo1984** [comment]
[PR #858](https://github.com/bolt/docs/pull/858)

**Pull request merged**